jest.config.js

```diff
@@ -9,8 +9,8 @@ module.exports = {
   moduleNameMapper: {
     "\\.(css)$": "<rootDir>/src/__mocks__/styleMock.js",
     "schema.js": "<rootDir>/src/__mocks__/schemaMock.js",
-    "logging.service": "<rootDir>/src/__mocks__/logging.service.js",
```

Comment:
I have a feeling I should remeber why this occured but you'll have to refresh my memory

Reply:
jest.unmock(path) doesn't work anymore in jest v24, so we can't mock out the entire service as it needs to be tested too. Instead we're mocking winston.